hack/ci/e2e.sh

```diff
@@ -187,9 +187,30 @@ EOF
         | grep -cv "node-role.kubernetes.io/master" \
     )"
 
-    # ginkgo regexes
-    SKIP="${SKIP:-}"
-    FOCUS="${FOCUS:-"\\[Conformance\\]"}"
+    # ginkgo regexes, use CI k/k presubmit regex by default
+    SKIP="${SKIP:-"\\[Slow\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\]"}"
+    FOCUS="${FOCUS:-""}"
+
+    # skip known issues
+    # [sig-node] RuntimeClass should run a Pod requesting a RuntimeClass with a configured handler [NodeFeature:RuntimeHandler]
+    # https://github.com/kubernetes/kubernetes/issues/80098
+    SKIP="RuntimeClass.should.run.a.Pod.requesting.a.RuntimeClass.with.a.configured.handler|${SKIP}"
+    # [sig-storage] In-tree Volumes [Driver: nfs]
+    # [sig-storage] PersistentVolumes NFS
+    # TODO: kind nodes can't mount nfs volumes: install apt install -y nfs-common in the base image, ...??
+    SKIP="In-tree.Volumes.\\[Driver:.nfs\\]|PersistentVolumes.NFS|${SKIP}"
```

Comment:
seems NFS can´t export an overlay fs https://lwn.net/Articles/743275/

Reply:
available since kernel 4.16 https://www.phoronix.com/scan.php?page=news_item&px=OverlayFS-NFS-Export-Linux-4.16

Reply:
it may be a bit before our hosts have this, let's put a TODO to try this when the host kernels are new enough. pretty sure our ubuntu GKE nodes are ~4.15